### PR TITLE
Limit SMBIOS reads to 0x000f0000-0x00100000

### PR DESCRIPTION
**Pull request by theopolis**

### CODE REVIEW

**Comment by osqueryer** [comment]
Build origin/pr/2135/merge (Job: 2735) failed :-1:

File: osquery/tables/system/linux/smbios_tables.cpp [hunk 1]
```diff
@@ -59,7 +59,17 @@ void LinuxSMBIOSParser::readFromSystab(const std::string& systab) {
       if (details.size() == 2 && details[1].size() > 2) {
         long long int address;
         safeStrtoll(details[1], 16, address);
-        readFromAddress(address, kLinuxSMBIOSRawLength_);
+        if (address < kLinuxSMBIOSRawAddress_ ||
+            address > (kLinuxSMBIOSRawLength_ + kLinuxSMBIOSRawLength_)) {
```
Comment (theopolis) [inline osquery/tables/system/linux/smbios_tables.cpp]: Length is repeated.

**Comment by osqueryer** [comment]
Build origin/pr/2135/merge (Job: 2736) failed :-1:

**Comment by theopolis** [comment]
Tested on a few 4.0 kernels booting EFI and works great!

**Pull request merged**